blueprints/jsonapi-resource/index.js

```diff
@@ -57,19 +57,16 @@ module.exports = {
     var modelOptions = merge({}, options, {
       entity: {
         name: entityName ? inflection.singularize(entityName) : ''
-      },
-      originBlueprintName: 'jsonapi-model'
+      }
     });
 
     var otherOptions = merge({}, options);
 
     return Promise.all([
       this._processBlueprint(type, 'jsonapi-model', modelOptions),
-      this._processBlueprint(type, 'addon-import', modelOptions),
```

Comment:
The addon-import blueprint is the one the creates the import of the add-on module in the app directory. I think it's necessary.